### ISSUE TITLE
How to get val loss in 3.x?

### ISSUE THREAD

**Issue by GeorgePearse** [BinItAI/visdet#706]
**Original issue:** https://github.com/open-mmlab/mmdetection/issues/9904
**Created:** 2023-03-08
**Last updated:** 2024-12-10

---

I have seen Validation Loss During Training #7971.
but there is no workflow in _base_/default_runtime.py.
my mmdetection version is 3.x
